### PR TITLE
Kbim 33 - MinIO Changes - Messages Changes

### PR DESCRIPTION
**Pull request by mnzit**

### CODE REVIEW

File: kogitoq/gadaq/src/main/java/life/genny/gadaq/cache/SearchCaching.java [hunk 1]
```diff
@@ -91,6 +96,22 @@ public void saveToCache() {
 		cacheDropdown("DEF_BUILDING",
 				new SearchEntity("SBE_SER_LNK_AREA_UNIT", "Area Unit Dropdown")
 						.setLinkValue("AREA_UNIT"));
+
+		// DEF_MESSAGE
+		cacheSearch(
+				new SearchEntity(SBE_MESSAGE, "Messages")
+						.add(new Filter(PRI_CODE, Operator.LIKE, "MSG_%"))
```
Comment (jasperRob) [inline kogitoq/gadaq/src/main/java/life/genny/gadaq/cache/SearchCaching.java]: Use STARTS_WITH and Prefix constant

File: bridge/pom.xml [hunk 2]
```diff
@@ -85,6 +85,22 @@
 			<artifactId>rest-assured</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- QwandaQ already has this but native build will fail so adding these here too-->
```
Comment (jasperRob) [inline bridge/pom.xml]: Have another look at this. See if you can find a way to remove it in all the child pom files.

**Review by jasperRob — COMMENTED** [review]
Address these comments.

File: kogitoq/gadaq/src/main/java/life/genny/gadaq/cache/SearchCaching.java (hunk 1 above)
Comment (mnzit) [reply kogitoq/gadaq/src/main/java/life/genny/gadaq/cache/SearchCaching.java]: Fixed

File: bridge/pom.xml (hunk 2 above)
Comment (mnzit) [reply bridge/pom.xml]: fixed

**Review by jasperRob — APPROVED** [review]
👍  Looking Good!

**Pull request merged**